docs/topics/vs-code-extension-interface.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id="vs-code-extension-interface_{context}"]
+= {ProductShortName} extension interface
+
+The interface of the {ProductName} ({ProductShortName}) extension is designed to make it easier for you to find information and perform actions:
+
+* In the left pane, you can see icons representing the applications analyzed by MTA, the rulesets used, and the issues discovered by the analysis. You can also click the report icon to open the MTA report in your browser.
```

Comment:
Do all these items really have icons? I thought only hints have icons. 

Aren't the applications displayed as a directory tree?